src/style.css

```
@@ -1 +1,83 @@
-/* Styles go here */
+ul,
+a,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav_list {
+  display: flex;
+}
+
+.nav_item {
```

```suggestion
.nav_item:not(:last-child) {
```
and then you can remove it below:
```
.nav_item:last-child {
  margin-right: 0;
}
```

---

++